pinot-query-planner/src/main/java/org/apache/pinot/query/planner/RelToStageConverter.java

```diff
@@ -24,16 +24,18 @@
 import java.util.stream.Collectors;
 import org.apache.calcite.rel.RelNode;
 import org.apache.calcite.rel.core.JoinRelType;
-import org.apache.calcite.rel.logical.LogicalCalc;
```

Comment:
Is LogicalCalc even useful ? It is essentially same functionality provided together by LogicalProject and LogicalFilter

Reply:
nope it is not used. I removed the calc rule so it will not be generated